### PR TITLE
[DOC] Add lesion masking during registration

### PR DESCRIPTION
**Pull request by danlurie**
Addresses https://github.com/poldracklab/fmriprep/issues/1110

### CODE REVIEW

**Comment by oesteban** [comment]
Hey @danlurie, can I ask you to work on a separate branch which name starts with `docs/`? Something like `docs/lesions`.

That will indicate circleCI that this is an only-docs build and save builds.

**Comment by danlurie** [comment]
Will do from now on!

File: docs/workflows.rst [hunk 1]
```diff
@@ -106,6 +106,16 @@ in a multiscale, mutual-information based, nonlinear registration scheme.
...
 
+When processing images from patients with focal brain lesions (e.g. stroke, tumor
+resection), it is possible to provide a lesion mask to be used during spatial
+normalization to MNI-space. ANTs will use this mask to minimize warping of 
+healthy tissue into damaged areas (or vice-versa). Lesion masks should be binary 
+NIfTI images (damaged areas = 1, everywhere else = 0) in the same space and
+resolution as the T1 image, and follow the naming convention specified in
+`BIDS Extension Proposal 3: Common Derivatives <https://docs.google.com/document/d/1Wwc4A6Mow4ZPPszDIWfCUCRNstn7d_zzaWPcfcHmgI4/edit#heading=h.9146wuepclkt>`_
+(e.g. ``sub-001_T1w_label-lesion_roi.nii.gz``). This file should be placed in
+the ``sub-*/anat`` directory of the BIDS dataset to be run through ``fmriprep``.
```
Comment (effigies) [inline docs/workflows.rst]: Can you start each new sentence on a new line? That makes future edits nicer to read on diffs.

(I haven't read this to edit, so if you choose to wait for more feedback before doing that, that's fine.)

File: docs/workflows.rst [hunk 2]
```diff
@@ -106,6 +106,17 @@ in a multiscale, mutual-information based, nonlinear registration scheme.
 In particular, spatial normalization is done using the `ICBM 2009c Nonlinear
 Asymmetric template (1×1×1mm) <http://nist.mni.mcgill.ca/?p=904>`_ [Fonov2011]_.
 
+When processing images from patients with focal brain lesions (e.g. stroke, tumor 
+resection), it is possible to provide a lesion mask to be used during spatial 
+normalization to MNI-space [Brett2011]_. ANTs will use this mask to minimize warping of
```
Comment (effigies) [inline docs/workflows.rst]: Brett2001

Comment (effigies) [reply docs/workflows.rst]: (Also new line.)

**Pull request merged**